docs/pages/home.js

```diff
@@ -0,0 +1,100 @@
+// @flow strict
+import { Box, Button, Flex, Heading, Text } from 'gestalt';
+import { type Node } from 'react';
+import HomePageCard from '../components/HomePageCard.js';
+import HomePageSection from '../components/HomePageSection.js';
+import Page from '../components/Page.js';
+import Hero from '../graphics/HeroGraphic.js';
```

Comment:
Should we lazy-load this and the SVGs? Doesn't seem like a blocker to me for this PR, but a possible future improvement